packages/paste-website/src/components/shortcodes/table-of-contents/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import {Anchor} from '@twilio-paste/anchor';
+import {TableOfContentsProps} from './types';
+import {StyledWrapper, StyledScrollSpy, StyledListItem} from './styles';
+import {slugify} from '../../../utils/RouteUtils';
+
+const TableOfContents: React.FC<TableOfContentsProps> = data => {
+  if (data == null || data.data == null) {
+    return null;
+  }
+
+  const headings = data.data;
```

Comment:
`data.data` is kind of weird.  Does the first `data` have a more accurate name by any chance?

Reply:
This is props, right?